forena.report.inc

```diff
@@ -1324,7 +1325,8 @@ function forena_report_parameters_form($form, &$form_state, $report_name) {
     $ctl['weight'] = array(
     	'#type' => 'weight',
       '#title' => 'Weight',
-      '#default_value' => $i,
+      '#default_value' => $i * 2 - $parameter_count + 1,
```

Comment:
These changes don't belong in this pull request.  It is carryover from the issue of weights being lost due to a default `#delta` of 10.